oidb/pepper/common/src/lib.rs

```diff
@@ -0,0 +1,89 @@
+// Copyright © Aptos Foundation
+
+use crate::asymmetric_encryption::{AsymmetricEncryption, elgamal_curve25519_aes256_gcm};
+use aes_gcm::aead::rand_core::{CryptoRng as AeadCryptoRng, RngCore as AeadRngCore};
+use anyhow::bail;
+use curve25519_dalek::digest::Digest;
+use rand_core::{CryptoRng, RngCore};
+use serde::{Deserialize, Serialize};
+use crate::asymmetric_encryption::elgamal_curve25519_aes256_gcm::ElGamalCurve25519Aes256Gcm;
+
+pub mod asymmetric_encryption;
+pub mod elgamal;
+pub mod jwt;
+pub mod vuf;
+
+pub fn sha3_256(input: &[u8]) -> Vec<u8> {
+    let mut hasher = sha3::Sha3_256::new();
+    hasher.update(input);
+    hasher.finalize().to_vec()
+}
+
+/// The spec of a request to this pepper service.
+#[derive(Debug, Deserialize, Serialize)]
+pub struct PepperRequest {
+    pub jwt: String,
+    /// If specified, generate pepper for `jwk.payload.iss, jwk.payload.sub, overriding_aud`.
+    /// Otherwise, generate pepper for `jwk.payload.iss, jwk.payload.sub, jwk.payload.aud`.
+    pub overriding_aud: Option<String>,
+    pub epk_hex_string: String,
+    pub epk_expiry_time_secs: u64,
+    pub epk_blinder_hex_string: String,
+    pub uid_key: Option<String>,
+}
+
+/// The response to `PepperRequestV0`, which contains the calculated pepper (hexlified) or a processing error.
+#[derive(Debug, Deserialize, Serialize)]
```

Comment:
Please add comments and explain these fields.

Not sure why there are two things here.

There is just one pepper.